### PR TITLE
app-misc/ddccontrol: Fix call to undeclared library function malloc

### PR DESCRIPTION
**Pull request by listout**
And some other fixes

Closes: https://bugs.gentoo.org/923582

### CODE REVIEW

File: app-misc/ddccontrol/ddccontrol-1.0.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+inherit autotools
+
+DESCRIPTION="Control monitor parameters, like brightness, contrast, RGB color levels via DDC"
+HOMEPAGE="https://github.com/ddccontrol/ddccontrol/"
+SRC_URI="https://github.com/ddccontrol/ddccontrol/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="doc gtk nls +pci static-libs"
```
Comment (juippis) [inline app-misc/ddccontrol/ddccontrol-1.0.3.ebuild:15]: Do we still need static-libs? Does something depend on it?

File: app-misc/ddccontrol/ddccontrol-1.0.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+# Upstream doesn't seem to care about tests: failures for lack of translations,
+# and no real test targets.
+RESTRICT='test'
```
Comment (juippis) [inline app-misc/ddccontrol/ddccontrol-1.0.3.ebuild:19]: Is this still the case?

File: app-misc/ddccontrol/ddccontrol-1.0.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+	touch config.rpath ABOUT-NLS
+	eautoreconf
+	intltoolize --force || die "intltoolize failed"
+}
+
+src_configure() {
+	# amdadl broken, bug #527268
+	econf \
+		--htmldir='$(datarootdir)'/doc/${PF}/html \
+		--disable-gnome-applet \
```
Comment (juippis) [inline app-misc/ddccontrol/ddccontrol-1.0.3.ebuild:77]:
```
 * QA Notice: Unrecognized configure options:
 * 
 *      --disable-gnome-applet
```

File: app-misc/ddccontrol/ddccontrol-1.0.3.ebuild [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+		--disable-amdadl \
+		$(use_enable doc) \
+		$(use_enable gtk gnome) \
+		$(use_enable nls) \
+		$(use_enable pci ddcpci) \
+		$(use_enable static-libs static)
```
Comment (juippis) [inline app-misc/ddccontrol/ddccontrol-1.0.3.ebuild:83]: Array-form is always preferred.

**Review by juippis — COMMENTED** [review]
`+gtk` gives me:
```
 * QA Notice: new icons were found installed but icon cache
 * has not been updated:
 *   /usr/share/icons/hicolor/48x48/apps/gddccontrol.png
 * Please make sure to call xdg_icon_cache_update()
 * in pkg_postinst() and pkg_postrm() phases of appropriate pkgs.
```

**Comment by arkamar** [comment]
Obsoleted by commit f72870d9ad8e75d818224f321ffaa1e7d0c7ae1a

**Pull request closed without merging**